src/shogun/regression/KRRNystrom.cpp

```diff
@@ -79,6 +79,14 @@ bool CKRRNystrom::solve_krr_system()
 {
 	int32_t n=kernel->get_num_vec_lhs();
 
+	if (n < m_num_rkhs_basis)
+	{
+		SG_SWARNING("Number of sampled rows (%d) must be less than or equal to "
```

Comment:
I see ...
But the test is not very meaningful then, as KRR's point is to use less than all.
Maybe change the default to something like "half of the data" if m=0?